client/src/components/Donation/components/StripeCardForm.js

```diff
@@ -1,10 +1,6 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import {
-  CardNumberElement,
-  CardExpiryElement,
-  CardCVCElement
```

Comment:
I think we'd be better of including this, to reduce the chance that a bank rejects the charge.  If the concern is that it's taking up too much space, why not put it on the same row as 'Your Card Expiration Month:'?  That's currently taking a full row for 5 characters.

Something like this:
![image](https://user-images.githubusercontent.com/15801806/70612288-f55b3c00-1c06-11ea-8565-5c3414ef7f91.png)

Reply:
We can make space by using 'Expiration Date:' or 'Expiration Month:' since it's clear from 'Your Card Number' what's going on.